### PR TITLE
Implemented links in slides

### PR DESCRIPTION
**Pull request by andresgnlez**

### CODE REVIEW

**Comment by tiagojsag** [comment]
@andresgnlez I added the link to the LDF slide, but it's missing some styling. can you take care of it, please?
thanks

File: app/src/components/Home/CoverPage.jsx [hunk 1]
```diff
@@ -73,6 +82,13 @@ class CoverPage extends Component {
           <p>
             {coverPageEntry.subtitle}
           </p>
+          {coverPageEntry.linkHref && coverPageEntry.linkText && <Rhombus direction="-right" color="-white">
+            <Link
+              to={coverPageEntry.linkHref}
+            >
+              {coverPageEntry.linkText}
+            </Link>
+          </Rhombus>}
```
Comment (nerik) [inline app/src/components/Home/CoverPage.jsx]: Please abstract to a method.

File: app/styles/components/c-cover-page.scss [hunk 2]
```diff
@@ -22,6 +22,12 @@
 
     @media #{$mq-tablet} {
       max-width: 870px;
+      min-height: 756px;
+    }
+
+    :global .c-rhombus {
```
Comment (nerik) [inline app/styles/components/c-cover-page.scss]: Is this going to affect all rhombus elements on the whole app?

File: app/src/components/Home/CoverPage.jsx (hunk 1 above)
Comment (andresgnlez) [inline app/src/components/Home/CoverPage.jsx]: done!!

**Comment by andresgnlez** [comment]
@tiagojsag sorry I don't understand you, could you give me some picture of it?

**Comment by clementprdhomme** [comment]
Don't forget to update the [task](https://www.pivotaltracker.com/story/show/129535275) when closed.

**Pull request merged**